docs/mining/proof-of-work/pool-mining/gominer/linux.md

```diff
@@ -0,0 +1,99 @@
+# **<i class="fa fa-linux"></i> *gominer* Linux Pool Mining User Guide**
+
+This guide is intended for basic initial setup of gominer on a mining pool of choice using a configuration file, more advanced options are available.
```

Comment:
Comma splice.  The second part (more advanced options) should be a separate sentence or separate with a semicolon.